users/konstantin/rules.mk

```diff
@@ -1,15 +1,12 @@
-BOOTMAGIC_ENABLE   = no
-COMMAND_ENABLE     = yes
-CONSOLE_ENABLE     = yes
-EXTRAKEY_ENABLE    = yes
-MOUSEKEY_ENABLE    = yes
-NKRO_ENABLE        = yes
-TAP_DANCE_ENABLE   = yes
-UNICODE_ENABLE     = yes
-
 SRC += konstantin.c
+ifneq (,$(filter yes,$(RGBLIGHT_ENABLE) $(RGB_MATRIX_ENABLE)))  # if either is yes
```

Comment:
Just a heads up, `RGB_MATRIX_ENABLE` may not always be set to "yes". 

`IS31FL3731` and `IS31FL3733` are valid settings for this, as well.

Reply:
Oh, thanks, I didn't know that. I'll try to find another way of writing this.

Reply:
For now, this should be okay.  most boards are using "yes", I think.